### DISCUSSION TITLE
How to deal with failing pipelines on the main branch?

### DISCUSSION THREAD

**Discussion by cveld** [Azure/AzOps#500] [Q&A] [answered]
Theoretically, when you have a large chunk (or any size for that matter) of deployment, and something fails either intermittently or structurally, what would be the procedure to remediate or repair?

**Comment by daltondhcp** [comment, marked as answer]
Obviously it depends on the nature of the problem, but in general it shouldn't ever make it to main if it is failing, i.e. we should catch the problem during PR testing. What is the scenario more specifically?

**Comment by cveld** [comment]
WhatIf does not execute the real thing. Only when the real thing happens
you will get real feedback. E.g. deploying a virtual machine.

Op wo 22 dec. 2021 08:35 schreef Johan Dahlbom ***@***.***>:
